File: main/src/main/java/net/fwitz/math/main/binary/dual/functions/trig/DualTanZ.java
package net.fwitz.math.main.binary.dual.functions.trig;

import net.fwitz.math.binary.dual.DualNumber;
import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot;

public class DualTanZ {
    public static void main(String[] args) {
        new DualNumberFunctionPlot("tan(z)")
                .computeFn(DualNumber::tan)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/dual/functions/trig/DualTanZ.kt
package net.fwitz.math.main.binary.dual.functions.trig

import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot

object DualTanZ {
    @JvmStatic
    fun main(args: Array<String>) = DualNumberFunctionPlot("tan(z)")
        .computeFn { z -> z.tan }
        .render()
}